randovania/games/prime1/json_data/Phendrana Drifts.txt

```diff
@@ -2397,6 +2426,19 @@ Extra - aabb: [329.07877, -1248.0734, -17.693329, 398.72052, -1143.9733, 42.7440
                       Slope Jump (Advanced)
                   Scan Visor and Combat/Scan Dash (Advanced)
           Shoot Plasma Beam and Use Grapple Beam
+          All of the following:
+              After Gravity Chamber Item (Lower)
+              Any of the following:
+                  All of the following:
+                      Charge Beam
```

Comment:
Charge Beam item does not always mean the player can shoot it. See the templates.